hw08_envdir_tool/main.go

```diff
@@ -1,5 +1,29 @@
 package main
 
+import (
+	"fmt"
+	"os"
+)
+
+const exitCode = 111
```

Comment:
А что означает этот exitCode? Не найду в списке https://unix.stackexchange.com/a/254747

Reply:
Взял из описания работы envdir.
"envdir exits 111 if it has trouble reading d, if it runs out of memory for environment variables, or if it cannot run child.  Otherwise its exit code is the same as that of child."